packages/common/src/utils/timeFrames.ts

```diff
@@ -242,103 +242,123 @@ type TimeFrameConfig = {
         originalSql: string,
         type: DimensionType,
     ) => string;
+    getAxisMinInterval: () => number | null;
```

Comment:
why not making it optional ? it doesn't even make sense on !DATES (for now) . Could we use it in the future for non dates ?